### PR TITLE
weave socket addtions.

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: roles/cadvisor/tasks/main.yml [hunk 1]
```diff
@@ -28,13 +25,13 @@
...
-#  sudo: yes
-#  template:
-#   src: cadvisor-consul.j2
-#    dest: "{{ cadvisor_consul_dir }}/cadvisor.json"
-#  notify:
-#    - restart consul
-#  when: cadvisor_enabled
-#  tags:
-#    - cadvisor
+- name: Set cadvisor consul service definition
```
Comment (tayzlor) [inline roles/cadvisor/tasks/main.yml]: I've added a better cadvisor health check in https://github.com/Capgemini/Apollo/pull/598/files might want to combine these PRs

File: roles/registrator/templates/registrator.service.j2 [hunk 2]
```diff
@@ -7,12 +7,14 @@ Requires=docker.service
...
 EnvironmentFile=/etc/environment
+Environment="DOCKER_HOST={{ registrator_docker_socket }}"
+
 ExecStartPre=-/usr/bin/docker kill registrator
 ExecStartPre=-/usr/bin/docker rm registrator
 ExecStartPre=/usr/bin/docker pull {{ registrator_image }}
 
 ExecStart=/usr/bin/bash -c "/usr/bin/docker run --rm --name registrator \
--v "{{ registrator_docker_socket }}:/tmp/docker.sock" \
+-v /var/run/weave/weave.sock:/tmp/docker.sock \
```
Comment (tayzlor) [inline roles/registrator/templates/registrator.service.j2]: should this be ```registrator_docker_socket```

Comment (enxebre) [inline roles/registrator/templates/registrator.service.j2]: Not really we just actually need better naming
registrator_docker_socket is the actual  socket address ->  = docker_host = unix://var/blah-blah ( used for environment variable)
What I want here is the path -> /var/blah-blah

File: roles/cadvisor/tasks/main.yml (hunk 1 above)
Comment (enxebre) [inline roles/cadvisor/tasks/main.yml]: cool, I'll just remove my cadvisor related changes from here

**Comment by tayzlor** [comment]
lgtm

**Pull request merged**